src/components/Username/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface UsernameProps {
+  username?: string;
```

Comment:
I think this field should be mandatory, because it is used to build a unique profile about you the user specifically.